_includes/contribute.html

```diff
@@ -1,4 +1,7 @@
 <p class="improve">
 	<a class="btn btn-outline" target="_blank" href="https://github.com/ambanum/disinformation-encyclopedia/edit/master/{{ page.path }}">&#9998; Improve this content</a>
 	<a class="btn btn-outline btn-quiet" target="_blank" href="/encyclopedia/contribute">How to contribute</a>
+	<a id="license-by-sa" target="_blank" href="/encyclopedia/license">
+		<img src="/assets/img/by-sa.svg" alt="">
```

Comment:
This should have an alt content  😉 
```suggestion
		<img src="/assets/img/by-sa.svg" alt="Creative Commons Attribution Share alike license">
```